tests/core/data/test_data_model.py

```diff
@@ -14,80 +14,16 @@
 
 
 class TestDataModel:
-    def test_deprecated_properties(self):
```

Comment:
I would revert this and rename the test so we know we are testing `last_edition_date`.